src/docs/codelabs/implicit-animations.md

```diff
@@ -0,0 +1,472 @@
+     return Column(children: <Widget>[
+       Image.network(owl_url),
+@@ -20,6 +21,7 @@
+           ),
+           onPressed: () => null),
+       AnimatedOpacity(
++          opacity: opacity,
+           child: Column(
+             children: <Widget>[
+               Text('Type: Owl'),
+```
+
+**4.** In addition to an `opacity` parameter, `AnimatedOpacity` requires a
+`duration` to use for its animation. For this example, you can start with 2 seconds:
+
+<?code-excerpt "opacity{3,4}/lib/main.dart"?>
+```diff
+--- opacity3/lib/main.dart
++++ opacity4/lib/main.dart
+@@ -21,6 +21,7 @@
+           ),
+           onPressed: () => null),
+       AnimatedOpacity(
++          duration: Duration(seconds: 2),
+           opacity: opacity,
+           child: Column(
+             children: <Widget>[
+```
+
+**5.** Finally, configure the animation to trigger when the user clicks the
+**Show details** button. To do this, change `opacity` state using
+the `onPressed()` handler for `MaterialButton`. To make the `FadeInDemo` widget become fully
+visible when the user clicks the **Show details** button, set `opacity`
+to 1:
+
+<?code-excerpt "opacity{4,5}/lib/main.dart"?>
+```diff
+--- opacity4/lib/main.dart
++++ opacity5/lib/main.dart
+@@ -19,7 +19,9 @@
+             'Show Details',
+             style: TextStyle(color: Colors.blueAccent),
+           ),
+-          onPressed: () => null),
++          onPressed: () => setState((){
++            opacity = 1;
++          })),
+       AnimatedOpacity(
+           duration: Duration(seconds: 2),
+           opacity: opacity,
+```
+
+{{site.alert.secondary}}
+  Notice that you only need to set the beginning and ending values of `opacity`.
+  The `AnimatedOpacity` widget manages everything in between.
+{{site.alert.end}}
+
+Here's the example with the completed changes you've made&mdash;run this
+example and click the **Show details** button to trigger the animation.
+
+<!-- AnimatedOpacity https://gist.github.com/4207fea3975b2d329e81d9c9ba84d271 -->
+<iframe
+  src="{{site.custom.dartpadx.embed-flutter-prefix}}?id=4207fea3975b2d329e81d9c9ba84d271"
+  style="border: 1px solid lightgrey; margin-top: 10px; margin-bottom: 25px"
+  frameborder="no" height="500" width="100%"
+></iframe>
+
+### Putting it all together
+
+The previous example demonstrates useful features of `AnimatedOpacity`:
+- `AnimatedOpacity` listens for state changes in its `opacity` property.
+- Whenever `opacity` changes, `AnimatedOpacity` automatically animates the
+widget's transition to the new value for `opacity`.
+- `AnimatedOpacity` requires a `duration` parameter to define the time it takes
+to animate the transition between an old `opacity` value and a new one.
+
+The previous example also uses a common workflow for using implicit animations:
+- First, pick a widget property to animate.
+- Next, choose an implicit animation that can animate that property.
+- Set the start and end values for the property that you're animating.
```

Comment:
This sounds kinda odd. You're really only picking a start value. In the last bullet point, where you trigger the animation you basically trigger it by picking the end value.

Reply:
Yes, I think you're right -- `Set the start and end values` is misleading. The "workflow" I'm summarizing here is essentially a series of decisions made at design time, so `Choose the start and end values...` would be the better phrase. WDYT?